### ISSUE TITLE
Output fails when targets matches a single file

### ISSUE THREAD

**Issue by avylove** [brandtbucher/specialist#2]
Running something like
```console
python -m specialist --output specialist --targets example/__init__.py example/__init__.py
```
fails with
```console
  File "/lib64/python3.11/site-packages/specialist.py", line 443, in <genexpr>
    (path_, output / path_.relative_to(common).with_suffix(".html"))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/pathlib.py", line 694, in with_suffix
    raise ValueError("%r has an empty name" % (self,))
ValueError: PosixPath('.') has an empty name
```

It seems the problem is in
```python
                common = pathlib.Path(
                    os.path.commonpath(paths)  # pylint: disable = no-member
                ).resolve()
```
This assumes there is more than one file and `common` will end up as a directory.

Adding this right below seems to fix it
```python
                if not common.is_dir():
                    common = common.parent
```

An alternative method would be to check the length of `paths`.

**Comment by brandtbucher** [comment]
Yep, nice catch! Would you like to open a PR?

**Comment by brandtbucher** [comment]
Fixed in 465ed94914bacd01989326d7b2391b1c6698805e.

**Comment by brandtbucher** [comment]
Shipped in `0.2.1`.

**Comment by avylove** [comment]
Awesome! Thanks!
